src/main/java/org/mutabilitydetector/config/JdkConfiguration.java

```diff
@@ -41,24 +55,52 @@
  * @see BigDecimal
  * @see BigInteger
  * @see Class
+ * @see Duration
+ * @see Instant
+ * @see LocalDate
+ * @see LocalDateTime
+ * @see LocalTime
+ * @see MonthDay
+ * @see OffsetDateTime
+ * @see OffsetTime
+ * @see Period
+ * @see Year
+ * @see YearMonth
+ * @see ZoneOffset
+ * @see ZonedDateTime
+ * @see DateTimeFormatter
  */
 public class JdkConfiguration extends ConfigurationBuilder {
-    @Override
-    public void configure() {
-        hardcodeAsDefinitelyImmutable(String.class);
-        hardcodeAsDefinitelyImmutable(Boolean.class);
-        hardcodeAsDefinitelyImmutable(Byte.class);
-        hardcodeAsDefinitelyImmutable(Character.class);
-        hardcodeAsDefinitelyImmutable(Short.class);
-        hardcodeAsDefinitelyImmutable(Integer.class);
-        hardcodeAsDefinitelyImmutable(Long.class);
-        hardcodeAsDefinitelyImmutable(Float.class);
-        hardcodeAsDefinitelyImmutable(Double.class);
-        hardcodeAsDefinitelyImmutable(BigDecimal.class);
-        hardcodeAsDefinitelyImmutable(BigInteger.class);
-        hardcodeAsDefinitelyImmutable(Class.class);
-        hardcodeAsDefinitelyImmutable(URI.class);
+	@Override
+	public void configure() {
+		hardcodeAsDefinitelyImmutable(String.class);
+		hardcodeAsDefinitelyImmutable(Boolean.class);
+		hardcodeAsDefinitelyImmutable(Byte.class);
+		hardcodeAsDefinitelyImmutable(Character.class);
+		hardcodeAsDefinitelyImmutable(Short.class);
+		hardcodeAsDefinitelyImmutable(Integer.class);
+		hardcodeAsDefinitelyImmutable(Long.class);
+		hardcodeAsDefinitelyImmutable(Float.class);
+		hardcodeAsDefinitelyImmutable(Double.class);
+		hardcodeAsDefinitelyImmutable(BigDecimal.class);
+		hardcodeAsDefinitelyImmutable(BigInteger.class);
+		hardcodeAsDefinitelyImmutable(Class.class);
+		hardcodeAsDefinitelyImmutable(URI.class);
+		hardcodeAsDefinitelyImmutable(Duration.class);
```

Comment:
```suggestion
		hardcodeAsDefinitelyImmutable("java.time.Duration");
```

And the same change for other lines.

Reply:
@remal what leads you to believe they need to be quoted as fully qualified class names?

Reply:
Hey Remal,
There is an overridden version hardcodeAsDefinitelyImmutable(Class<?> immutableClass), which is being used the JdkConfiguration class.